style.css

```diff
@@ -5,6 +5,7 @@ html, body {
     height: 100%;
     background: #222;
     overflow: hidden;
+    user-select: none;
```

Comment:
I'm a little nervous about applying this style to the entire body.  Is there nowhere a user might want to select any text?  There are some color pickers on the GUI, one could imaging a desire to copy-and-paste color values there.  Can we be more selective with how this style is applied?

Reply:
You are right.

Actually `user-select` won't influence input elements, which means user still can select, copy-and-paste **values** in color picker of control panel, only **labels** is unselectable, which means the color picker of bottom right is not selectable anymore, that would be a problem.

Seems like we got two choices:
 - Add `user-select: text` to **`.pixelPicker`**
 - Move `user-select: none` from **`body`** to **`#pbrMath, #inpututs`**

Reply:
Whichever seems best, just not on the body/html.  Thanks!